recipes/qwt/all/conanfile.py

```diff
@@ -87,8 +87,15 @@ def _patch_qwt_config_files(self):
             qwtbuild += "CONFIG += force_debug_info\n"
         tools.save(qwtbuild_path, qwtbuild)
 
+    def _patch_qmake_generator_files(self):
+        # Work around the qmake generator bug (https://github.com/conan-io/conan/pull/9568) which
```

Comment:
The PR is now merged. How fast are you able to update your Conan client(s), when it will be released?

Since you are not yet an approved CCI contributor, it would take at least until next Monday before this gets merged

Reply:
Some weeks. We usually wait for a minor (fix) version.

Reply:
Seems like this was released? https://github.com/conan-io/conan/releases/tag/1.40.1

Reply:
@prince-chrismc looks to be correct.
`_patch_qmake_generator_files` will not break the new 1.40.1, and will allow this revision to work for previous conan versions as well. If that isn't desirable, we can cancel this PR.

Reply:
If we do ditch this change, we should replace it with:
`required_conan_version = ">=1.40.1"`